x/stake/client/cli/tx.go

```diff
@@ -95,7 +97,12 @@ func GetCmdCreateValidator(cdc *wire.Codec) *cobra.Command {
 func GetCmdEditValidator(cdc *wire.Codec) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "edit-validator",
-		Short: "edit and existing validator account",
+		Short: "edit an existing validator account",
+		Long: strings.TrimSpace(`
+Edits an existing validator account. All flags are required. Example:
+
+gaiacli stake edit-validator --moniker "TestMoniker" --identity "identity..." --website "https://www.example.com/" --details "This is a test validator."
```

Comment:
Like this example. We should keep the example with all of the flags just to demonstrate.